### PR TITLE
[SDK-10327] Captions best practices app

### PR DESCRIPTION
**Pull request by david-almaguer**
This Captions best practices app demonstrates the following functionality:
1. Listen for caption events
2. Disable caption rendering
3. Display captions in a different view.

### CODE REVIEW

File: JWBestPracticeApps/Captions/Captions/PlayerViewController.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+/**
+ The `PlayerViewController` handles the caption events from the player and forward them through the `PlayerCaptionsDelegate`.
+ */
+class PlayerViewController: JWPlayerViewController {
+    weak var captionsDelegate: PlayerCaptionsDelegate?
+    
+    override func jwplayer(_ player: JWPlayer, updatedCaptionList options: [JWMediaSelectionOption]) {
+        super.jwplayer(player, updatedCaptionList: options)
+        DispatchQueue.main.async { [weak self] in
```
Comment (DavidPerezP124) [inline JWBestPracticeApps/Captions/Captions/PlayerViewController.swift:28]: Overall looks good, the only question I have is if these dispatch calls should be on the JWPVC subclass or the `ViewController`?. It is cleaner if we keep it here, but the object that benefits from these being called on the main thread is the one that will update the UI elements.

Comment (david-almaguer) [reply JWBestPracticeApps/Captions/Captions/PlayerViewController.swift:28]: Good point, I will make the change.

**Comment by DavidPerezP124** [comment]
It looks good, and tested. Only issue I have is with installing cocoapods it produces an error when installing the pods.
I had to update the project format for the Captions project to 13.0 compatible. I'm on 14.0 so probably a cocoapods issue.

Screen Shot 2022-11-29 at 12 59 40

**Review by Tozarf — APPROVED** [review]
LGTM 👍

**Review by s-seibert — APPROVED** [review]

**Review by IvanEnroute — APPROVED** [review]
LGTM

**Pull request merged**